app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new # Guest user (not logged in)
+
+    if user.admin?
```

Comment:
- [ ] @Team great work implementing the change requested, however, you shifted from using the string `admin` to `admin?` method, which is not defined in your app leading to the error below when I try to access the posts index page, you can solve this by adding the admin? the method in your user model 
```
 def admin?
     role == 'admin' ? true : false
 end
````
![image](https://github.com/ehmaddd/blog_app/assets/61408860/2e3cce49-2a8e-4af8-b1d2-74bc640e5dbc)

Reply:
Added admin? method as per instructions